grad-check.lua

```diff
@@ -0,0 +1,148 @@
+--[[
```

Comment:
Is this file used? I don't think it is. Seems like a copy of the example from rnn repo.